modules/project_cleanup/main.tf

```diff
@@ -52,14 +53,17 @@ module "scheduled_project_cleaner" {
   topic_name                     = var.topic_name
   function_available_memory_mb   = 128
   function_description           = "Clean up GCP projects older than ${var.max_project_age_in_hours} hours matching particular tags"
-  function_runtime               = "go113"
+  function_runtime               = "go121"
```

Comment:
suggestion: Do we want to also bump https://github.com/terraform-google-modules/terraform-google-scheduled-function/blob/master/modules/project_cleanup/function_source/go.mod#L3 to go `1.21`?  Might also need a `mod tidy`.

Reply:
done

Reply:
Marked as a breaking change given the change in function runtime.  That said it's going into what is already a major release anyway.